nbs/15_gsa_gls/05-snps_into_chr_cov.ipynb

```diff
@@ -31,12 +31,14 @@
     "tags": []
    },
    "source": [
+    "(Please, take a look at the README.md file in this directory for instructions on how to run this notebook)\n",
+    "\n",
     "This notebook computes the covariance of SNPs for each chr."
```

Comment:
It's totally fair if you don't want to have the explanation of how to run the notebook in each output file for each dataset, but I would expand the description of what this file is doing, since if should be the same for each dataset and would help a lot with comprehension of the notebook.

Reply:
Thank you. I updated the notebooks' description.